lib/cisco_os_shim/core/version.rb

```diff
@@ -18,5 +18,5 @@ module Cisco
 
 # Add version number to Cisco::Shim namespace
 module Cisco::Shim
-  VERSION = '1.1.0'
+  VERSION = '1.0.0-dev'
```

Comment:
Are we going to bump the version down to 1.0 and release a new repo called cisco_os_shim?  The current repo name might cause confusion.